bin/dfx-sns-demo-mksns

```diff
@@ -15,12 +15,13 @@ source "$SOURCE_DIR/clap.bash"
 # Define options
 clap.define short=n long=network desc="The dfx network to use" variable=DFX_NETWORK default="local"
 clap.define long=confirmation_text desc="A text that needs to be confirmed by the participants" variable=CONFIRMATION_TEXT default=""
+clap.define short=o long=index-offset desc="The initial value for the dfx-sns-config-random index parameter" variable=INDEX_OFFSET default="0"
```

Comment:
There is no "offset" here. This just is the index, isn't it?

Reply:
I thought it would be easier to understand when using the same parameter name, but I agree, it doesn’t make much sense.